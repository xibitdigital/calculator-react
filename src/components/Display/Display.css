.display {
    background-color: #999;
    box-sizing: border-box;
    color: #ffffff;
    font-size: 50px;
    font-weight: lighter;
    height: 80px;
    padding: 20px 15px 5px 0;
    position: relative;
    text-align: right;
    width: 100%;
}
