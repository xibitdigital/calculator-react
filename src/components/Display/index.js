import React from 'react'

import './Display.css'

export default function Display({ value }) {
    return <div className="display">{value}</div>
}
