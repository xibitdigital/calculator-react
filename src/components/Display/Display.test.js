import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Display from '.'

test('Should render Display with value 10', () => {
    const display = shallow(<Display value={10} />)
    expect(shallowToJson(display)).toMatchSnapshot()
})
