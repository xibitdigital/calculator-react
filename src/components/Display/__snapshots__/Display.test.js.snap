// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Display with value 10 1`] = `
<div
  className="display"
>
  10
</div>
`;
